war_or_peace_runner.rb

```diff
@@ -0,0 +1,50 @@
+# not required
+require './lib/card'
+require './lib/deck'
+require './lib/turn'
+require './lib/player'
+require 'pry'
+# required
+require './lib/game'
+
+# all_cards = game.create_full_deck
+# all_cards.shuffle!
+
+# player1 = Player.new('Megan', deck1)
+# player2 = Player.new('Aurora', deck2)
+
+turn = Turn.new(player1, player2)
+game = Game.new(turn)
+
+game.start(player1, player2)
```

Comment:
Your `start` method doesn't take arguments so we don't need to pass players here. Your `initialized` method in `Game` does that for you.